deploy/charts/approver-policy/values.yaml

```diff
@@ -109,3 +109,6 @@ commonLabels: {}
 # -- Optional allow custom annotations to be placed on cert-manager-approver pod
 podAnnotations: {}
 
+crds:
+  # -- Whether or not to install the crds.
+  enabled: true
```

Comment:
This and the change to the CRD template file seems unrelated. Please revert and add it in a separate PR.

Reply:
see https://github.com/cert-manager/approver-policy/pull/316/files